src/pages/Onboard/AcceptInvitationPage.vue

```diff
@@ -0,0 +1,392 @@
+        return !this.invitationId
+      },
+      loadingKey: 'loadingKey',
+      pollInterval: 5000,
+      update: data => data.membership_invitation_by_pk
+    }
+  }
+}
+</script>
+
+<template>
+  <v-container
+    class="ma-0 pa-0 position-relative h-100 bg-grey overflow-y-hidden"
+    fluid
+  >
+    <transition-group name="fade" mode="out-in">
+      <div
+        v-if="$vuetify.breakpoint.smAndUp"
+        key="slash-1"
+        class="slash-blue o-slash slash-1"
+        :class="[slash1Class, $vuetify.breakpoint.smAndUp ? 'slash-1' : '']"
+      >
+      </div>
+      <div
+        key="slash-3"
+        class="slash-grey o-slash slash-3"
+        :class="[slash3Class, $vuetify.breakpoint.smAndUp ? 'slash-3' : '']"
+      >
+      </div>
+      <div
+        v-if="$vuetify.breakpoint.smAndUp"
+        key="slash-4"
+        class="slash-grey o-slash slash-4"
+        :class="slash4Class"
+      >
+      </div>
+      <div
+        v-if="$vuetify.breakpoint.smAndUp"
+        key="slash-5"
+        class="slash-orange o-slash slash-5"
+        :class="slash5Class"
+      >
+      </div>
+
+      <div v-if="$vuetify.breakpoint.smAndUp" key="name-team-slashes">
+        <div
+          v-for="i in noSlash"
+          :key="`name-team-slash-${i}`"
+          class="o-slash slash-2"
+          :class="[
+            slash2Class,
+            i % 2 === 0 ? 'slash-grey' : 'slash-grey-reverse'
+          ]"
+          :style="slash2Style(i)"
+        ></div>
+      </div>
+    </transition-group>
+    <v-container
+      v-if="!loadingPage"
+      class="position-absolute onboard-content text-center pa-0"
+      fluid
+    >
+      <v-row v-if="invitationError" align="center">
+        <v-col class="grey--text text--lighten-5 mx-12">
+          <div class="display-1">
+            <span> {{ mutationErrorMessage || errorMessage }} </span>
+          </div>
+          <div>
+            <v-btn class="mt-8" color="primary" x-large @click="toDashboard">
+              Back to the dashboard!
+              <v-icon right>fas fa-rocket</v-icon>
+            </v-btn>
+          </div>
+        </v-col>
+      </v-row>
+      <v-row v-else align="center">
+        <v-col>
+          <div class="display-1 grey--text text--lighten-5">
+            You've been invited to join {{ teamName }}!
+          </div>
+          <div class="subtitle grey--text text--lighten-5 mt-4">
+            To accept, please click below.
+          </div>
+          <div class="mt-8">
+            <v-btn
+              class="mr-3"
+              color="accentPink"
+              dark
+              depressed
+              :loading="loading"
+              @click="accept"
+            >
+              <v-icon class="pr-4">fa-user-friends</v-icon>
+              Accept
+            </v-btn>
+
+            <v-btn
+              outlined
+              class="white--text"
+              :loading="deleting"
+              @click="decline"
+            >
+              No Thanks
```

Comment:
Can we provide a confirmation when `No Thanks` is pressed? I can't think of many scenarios where this is the action the user intended to take and can see how it would be annoying if they accidentally pressed the wrong button.